impl/DTable.tcc

```diff
@@ -137,15 +140,43 @@ void DTable<InstType, AnnotationType, AnnotationTypeAllocator>::configure(const
                     tags = MatchSet<Tag>(inst["tags"].get<std::vector<std::string>>());
                 }
 
+                const bool is_expansion = inst.find("expand") != inst.end();
+
                 if ((inclusions.isEmpty() && exclusions.isEmpty()) || (inclusions.isEmpty() && tags.isEmpty())) {
                     // Inclusions & exclusions are empty, or inclusions empty and tags empty, no filtering active
-                    parseInstInfo_(jfile, inst, mnemonic, tags);
+                    if (!is_expansion)
+                    {
+                        parseInstInfo_(jfile, inst, mnemonic, tags);
+                    }
+                    else
+                    {
+                        expansions.emplace_back(std::bind(
+                            &DTable<InstType, AnnotationType, AnnotationTypeAllocator>::parseInstInfo_,
+                                                                                                  this,
+                                                                                                  jfile,
+                                                                                                  inst,
+                                                                                                  mnemonic,
+                                                                                                  tags));
```

Comment:
Since std::bind is making copies of all these arguments anyway, you could just as easily store them in a struct and avoid the std::function overhead.